devlab/tests/filtering_utils.py

```diff
@@ -61,6 +61,20 @@ def filter_images(self, src_data_list):
                     src_data_list.pop(index)
         return [src_data_list, popped_img_list]
 
+    def filter_tenants(self, src_data_list):
+        loaded_data = self.load_file('configs/filter.yaml')
+        filter_dict = loaded_data[0]
+        popped_ten_list = []
+        src_data_list = [x.__dict__ for x in src_data_list]
+        if 'tenants' not in filter_dict:
+            return [src_data_list, []]
+        for ten in src_data_list[:]:
```

Comment:
Also `ten` name for variable is misleading, it reads as "number 10", not "tenant", I would suggest renaming to `t`.